.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+# CI for pull requests
+# https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle?learn=continuous_integration&learnProduct=actions
+# Using gradle-build-action. See https://github.com/gradle/gradle-build-action
+
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        flavour: [assembleOculusvrArm64, assembleHvrArm64]
+
+    steps:
+      - name: Checkout Wolvic
+        uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - name: Checkout third parties
+        uses: actions/checkout@v2
+        with:
+          repository: Igalia/wolvic-third-parties
```

Comment:
We use this file to associate a Wolvic version with a third_party commit: https://github.com/Igalia/wolvic/blob/main/third_party_hash

Would it be possible to take the commit specified in that file?

Reply:
This can be done as a follow-up, main branch is fine for now

Reply:
OK, I guess it should be doable.